logs/archived/spring-boot-logger-2022-07-26.0.log

```diff
@@ -0,0 +1,180 @@
+2022-07-26 10:56:52,454 INFO org.springframework.boot.StartupInfoLogger [restartedMain] Starting PaketomatApplication using Java 17.0.3.1 on DESKTOP-2HFIQUK with PID 3244 (C:\Users\StefanTomic\Documents\paketomat\target\classes started by StefanTomic in C:\Users\StefanTomic\Documents\paketomat)
```

Comment:
Please delete this file.

Reply:
Done.

Reply:
I can still see this file, maybe you forgot to commit it.

Reply:
It was generated by the spring, i deleted it and after the other commits it was once more time generated